web/templates/popovers/navbar/navbar_personal_menu_popover.hbs

```diff
@@ -16,13 +16,7 @@
         </header>
         <section class="dropdown-menu-list-section personal-menu-actions" data-user-id="{{user_id}}">
             <ul role="menu" class="popover-menu-list">
-                {{#if user_time}}
-                <li role="none" class="text-item hidden-for-spectators popover-menu-list-item">
-                    <i class="popover-menu-icon zulip-icon zulip-icon-clock" aria-hidden="true"></i>
-                    {{#tr}}{user_time} local time{{/tr}}
-                </li>
-                <li role="separator" class="popover-menu-separator"></li>
-                {{/if}}
+
```

Comment:
Why do you have a blank line here?

Also, probably `user_time` should not be passed into the template.

Reply:
Okay I will do the redo the commit